packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -1,8 +1,8 @@
-/*
+/*-
  *
  * Hedera JSON RPC Relay
  *
- * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
```

Comment:
revert this and any other license header changes where you're changing an old header date into this year.
Might be a bug in an automated script